kn@0.17.rb

```diff
@@ -12,10 +12,10 @@ class KnAT017 < Formula
 
   if OS.mac?
     url "https://github.com/knative/client/releases/download/#{v}/kn-darwin-amd64"
-    sha256 "8f7b9f2dd11f2bcabdc89c50a4b865e7822cf1e815131b35435f2335199ee0d9"
+    sha256 "63c8cec08d4f5d9ea4c960c063813a679ccdca73409942b18c1a427d8ccf085e"
```

Comment:
please update the version to 0.17.4, too, in this file.

Reply:
fixed